src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double sideB;
+
+    public Rectangle(Colors color, double side, double sideB, double height) {
```

Comment:
Creating a figure, don't pass expressions in the constructor. Create separate variables and pass them on for better code readability.